### DISCUSSION TITLE
How to autostart an application without opening its window?

### DISCUSSION THREAD

**Discussion by otakutyrant** [i3/i3#5289] [Q&A] [answered]
I have some applications, like fcitx5, clash for windows, and fragments, to autostart. I put `exec --no-start-id fcitx5` and so on into the i3 config, however I have to close their windows to keep windows few every time after the autostart process, because I can open them again via their system trap.

**Comment by okraits** [comment, marked as answer]
With system trap, do you mean the system tray?

I think that i3 is not involved in that. i3 just manages the windows which are created by the applications. You have to check wether your applications support minimizing to tray upon start.

**Reply by otakutyrant** [reply]
Sorry for the typo. You are right. I go to convince applications developer to support the silent start feature.
